packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -132,7 +148,19 @@ function FilterMenu(props: TFilterMenuProps) {
   );
 
   return (
-    <Popover.Root defaultOpen={props.isDisabled ? false : props.defaultOpen}>
+    <Popover.Root
+      defaultOpen={props.isDisabled ? false : props.defaultOpen}
+      onOpenChange={(open) => {
```

Comment:
This removes the filter from being visible in the filters list if the menu is closed and no values have been selected